### ISSUE TITLE
createSession getting error in main.dart

### ISSUE THREAD

**Issue by Abhisheksharma654** [myConsciousness/atproto.dart#1793]
Don't create an issue for questions but let's use **[Discussions](https://github.com/myConsciousness/atproto.dart/discussions)**.
Is anyone know how to solve createSession error in main.dart.?

**Comment by myConsciousness** [comment]
Hi @Abhisheksharma654 !

I guess something wrong with parameters, can you share your code?

**Comment by Abhisheksharma654** [comment]
![image](https://github.com/user-attachments/assets/1867eeb8-6ba8-444b-84f8-21a4847b2b33)

**Comment by myConsciousness** [comment]
@Abhisheksharma654 

Did you add `bluesky` package to your `pubspec.yaml` and import it?

Also, we can no longer use the `bluesky_cards` package! This package is no longer maintained and is not compatible with the latest `bluesky` package. You can fork the `bluesky_cards` source, but you will need to refactor it to fit the latest Dart syntax.

**Comment by Abhisheksharma654** [comment]
![image](https://github.com/user-attachments/assets/86dfe7ad-22cc-49fc-8fa8-4a4e40a09c6c)
Bluesky package has already been added.

**Comment by Taolimon** [comment]
Hey there, I'm getting the same error.
I've tried following the bluseky tutorial on the atproto.dart website but it doesn't recognise the `.createSession` function.
I've tested it on both VSCode and Android Studio:
1. Create a new blank flutter app
2. Install the `bluesky` package using `flutter pub add bluesky` and `flutter pub get`
3. Import the package in the main.dart file.
4. Type the basic createSession code

It reproduces the error. I'm new to flutter and dart so I'm sure I might be missing a crucial step or configuration issue.

![Code_LbkAkMZpUv](https://github.com/user-attachments/assets/a3357200-bc0c-438a-a29b-a58c59489999)
![pubspecyaml](https://github.com/user-attachments/assets/d477301d-a85b-463e-8611-a495af111104)
